apis/cc-de_CH/atrium-api-cc-de_CH-common/src/main/kotlin/ch/tutteli/atrium/api/cc/de_CH/mapAssertions.kt

```diff
@@ -6,6 +6,15 @@ import ch.tutteli.atrium.creating.AssertionPlant
 import ch.tutteli.atrium.creating.AssertionPlantNullable
 import ch.tutteli.atrium.domain.builders.AssertImpl
 
+/**
+ * Makes the assertion that [AssertionPlant.subject]'s [Map] contains [key].
+ *
+ * @return This plant to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+fun <K, V> Assert<Map<K, V>>.enthaeltKey(key: K)
```

Comment:
Remove type parameter V as you don't use it within the function, use `Assert<Map<K, *>>` instead